core/src/main/java/org/fao/geonet/kernel/unpublish/UnpublishInvalidMetadataJob.java

```diff
@@ -134,7 +138,7 @@ public boolean matches(Object obj) {
         }
     };
 
-    public static final Filter ErrorFinder  = new Filter() {
+    public static final Filter ErrorFinder = new Filter() {
```

Comment:
static final should be uppercase and at the beginning of the class, but out of the pr scope.